### PR TITLE
Create PayablewithBankingEvent.sol

### LINKED ISSUE #63: Smart Contract for Simple Banking
### Is your feature request related to a problem? Please describe.
To build smart contract for simple banking

### Describe the solution you'd like
Write smart contract for depositing, withdrawal and authentication.

### PR DESCRIPTION
**Pull request by Gokul-Ks**
## Description

Created Solidity contract for simple banking processes <br />

Fixes #63 

## Developer's checklist

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-check on my work

**If changes are made in the code:**

- [x] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [x] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [x] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR

### CODE REVIEW

File: contracts/Misc/PayablewithBankingEvent.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.4;
```
Comment (advaita-saha) [inline contracts/Misc/PayablewithBankingEvent.sol:1]:
```suggestion
pragma solidity ^0.8.3;
```

**Review by advaita-saha — CHANGES_REQUESTED** [review]
Complete the mentioned changes

**Review by Gokul-Ks — COMMENTED** [review]
I have completed the requested change @advaita-saha

**Comment by advaita-saha** [comment]
Your contract is not getting compiled
https://github.com/metafy-social/web3-smart-contracts/actions/runs/3213673901/jobs/5253511613

Please fix these issues

**Pull request closed without merging**